stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/BackgroundDecorator.kt

```diff
@@ -68,18 +69,22 @@ internal class BackgroundDecorator : BaseDecorator() {
 
     override fun decoratePlainTextWithFileAttachmentsMessage(
         viewHolder: PlainTextWithFileAttachmentsViewHolder,
-        data: MessageListItem.MessageItem
+        data: MessageListItem.MessageItem,
     ) = decorateAttachmentsAndBackground(
         viewHolder.binding.backgroundView,
         viewHolder.binding.fileAttachmentsView,
         data
     )
 
     override fun decorateGiphyMessage(viewHolder: GiphyViewHolder, data: MessageListItem.MessageItem) {
-        viewHolder.binding.cardView.shapeAppearanceModel = ShapeAppearanceModel.builder()
+        viewHolder.binding.cardView.background = ShapeAppearanceModel.builder()
             .setAllCornerSizes(DEFAULT_CORNER_RADIUS)
             .setBottomRightCornerSize(SMALL_CARD_VIEW_CORNER_RADIUS)
             .build()
+            .let(::MaterialShapeDrawable)
+            .apply {
+                setTint(ContextCompat.getColor(viewHolder.context, R.color.stream_ui_background_default))
```

Comment:
We gave a `getColorCompat` extension for this

Reply:
Changed